lib/file-utilities.js

```diff
@@ -94,7 +94,7 @@ const getFiles = (config, dir, callback) => {
 
 const streamFiles = (config, logfiles, callback) => {
     logfiles.forEach((file) => {
-        var tail, i, labels;
+        let tail, i, labels;
```

Comment:
We don't need to forward declare `tail` and `i`. Let's just declare them when we need them.

Reply:
Done!